headers/IntPointData.h

```diff
@@ -51,21 +51,21 @@ class IntPointData
         {
             DebugStop();
         }
-        int nstate = coefs.size()/phi.size();
+        auto nstate = coefs.size()/phi.size();
```

Comment:
This feels like abuse of `auto` doesn't it? specially since it's supposed to be a teaching code...